crates/bevy_transform/src/components/global_transform.rs

```diff
@@ -211,6 +211,28 @@ impl GlobalTransform {
         self.0.translation
     }
 
+    /// Get the rotation as a [`Quat`].
+    ///
+    /// The transform is expected to be non-degenerate and without shearing, or the output will be invalid.
+    /// 
+    /// It is calculated using `to_scale_rotation_translation`, meaning that you
+    /// shoud probably use it directly if you also need translation or scale.
+    #[inline]
+    pub fn just_rotation(&self) -> Quat {
```

Comment:
This name seems kind of clunky to me. Is there a reason this can't just be `rotation`?

Reply:
I mentionned the reason in the PR message, but I'm not a 100% sure about it, so rotation() is fine with me too.

Reply:
Definitely prefer `rotation`.

Reply:
Maybe `to_scale` and `to_rotation` so it's clear these aren't simple accessor functions like `translation`?